source/_elephpants/2018-04-14-php-yorkshire.md

```diff
@@ -0,0 +1,10 @@
+---
+categories:
+    - PHP Yorkshire
```

Comment:
This should be a lower-cased, non-spaced string like `phpyorkshire`. It's used as an array key.